config/main.py

```diff
@@ -58,9 +59,11 @@
 CFG_LOOPBACK_NAME_TOTAL_LEN_MAX = 11
 CFG_LOOPBACK_ID_MAX_VAL = 999
 CFG_LOOPBACK_NO="<0-999>"
+PIPE_FILE = "/tmp/reload.txt"
 
 
 asic_type = None
+child_pid=0
```

Comment:
I'm not too keen on making this a global, since it is only used by `config reload`.

Also, should all spaces around `=`